eras/shelley/impl/src/Cardano/Ledger/Shelley/Genesis.hs

```diff
@@ -196,20 +209,33 @@ instance Era era => FromJSON (ShelleyGenesis era) where
     Aeson.withObject "ShelleyGenesis" $ \obj ->
       ShelleyGenesis
         <$> (forceUTCTime <$> obj .: "systemStart")
-        <*> obj .: "networkMagic"
-        <*> obj .: "networkId"
-        <*> obj .: "activeSlotsCoeff"
-        <*> obj .: "securityParam"
-        <*> obj .: "epochLength"
-        <*> obj .: "slotsPerKESPeriod"
-        <*> obj .: "maxKESEvolutions"
-        <*> obj .: "slotLength"
-        <*> obj .: "updateQuorum"
-        <*> obj .: "maxLovelaceSupply"
-        <*> obj .: "protocolParams"
+        <*> obj
+          .: "networkMagic"
+        <*> obj
+          .: "networkId"
+        <*> obj
+          .: "activeSlotsCoeff"
+        <*> obj
+          .: "securityParam"
+        <*> obj
+          .: "epochLength"
+        <*> obj
+          .: "slotsPerKESPeriod"
+        <*> obj
+          .: "maxKESEvolutions"
+        <*> obj
+          .: "slotLength"
+        <*> obj
+          .: "updateQuorum"
+        <*> obj
+          .: "maxLovelaceSupply"
+        <*> obj
+          .: "protocolParams"
```

Comment:
did you still want to collapse this?

Reply:
Yes, this was a formatting accident. Fixed